crates/sargon/src/profile/v100/networks/network/network_id.rs

```diff
@@ -140,23 +141,23 @@ impl NetworkID {
             Zabanet => ScryptoNetworkDefinition::zabanet(),
             Enkinet => ScryptoNetworkDefinition {
                 id: Enkinet.discriminant(),
-                logical_name: String::from("enkinet"),
-                hrp_suffix: String::from("tdx_21_"),
+                logical_name: Cow::Borrowed("enkinet"),
```

Comment:
`Cow::Borrowed` cool, enlight me! gives better performance?

Reply:
Well, not my choice, just aligning with the Scrypto model updates.

Reply:
perhaps something we should adopt in Sargon later on